yarn.lock

```diff
@@ -3154,6 +3154,11 @@
     cloudinary "^1.25.1"
     into-stream "^5.1.0"
 
+"@strapi/design-system@0.0.1-alpha.75":
+  version "0.0.1-alpha.75"
+  resolved "https://registry.yarnpkg.com/@strapi/design-system/-/design-system-0.0.1-alpha.75.tgz#8a06eb07a0d25791a93537578387e401dff116d2"
+  integrity sha512-JSIRFCiAOtTbjMDIa/KULSkfyPP/mphBhJDdeBx8bLNDKhU0SX3/zSXghjMWQ48aHvmyWTfwm8SVdw6EolzpIQ==
+
```

Comment:
🤔  I don't know

Reply:
I've moved the DS update to another PR: https://github.com/strapi/strapi/pull/12473 because many snapshots are going to be updated, so this shouldn't live in PRs :)